frontend/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
```

Comment:
Got thrown by this as well in https://github.com/gpuctl/gpuctl/pull/239. I *think* if you `npm update` then prettier will change it's mind and want the trailing commas gone.